README.md

```diff
@@ -27,23 +29,225 @@ You can use the package as follows:
+
+#### Amazon S3
+
+Amazon Simple Storage Service (Amazon S3) is an object storage service that offers scalability, data availability, security, and performance.
+
+We provide methods to easily list and delete objects inside buckets; read and write content within S3 objects; upload a dataframe into csv or parket format to a specific location; generate and upload an Amazon Quicksight manifest in S3 in order to create analysis in Amazon Quicksight, and so on.
+
+##### How to Upload an Object to S3
+
+```python
+from aws.s3_handler import S3Handler
+
+s3_handler = S3Handler()
+s3_handler.put_object(bucket="your_bucket_name", object_key="your_object_key", content="Your object content.")
+```
+
+#### AWS Secrets Manager
+
+AWS Secrets Manager protects secrets needed to access applications, services, and IT resources. The service allows rotating, managing, and retrieving credentials, keys, and other secrets.
+
+We provide methods to get the content of a previously created secret.
+
+##### Example of usage
+
+```python
+from aws.secrets_manager_handler import SecretsManagerHandler
+
+secrets_manager = SecretsManagerHandler()
+secret_content = secrets_manager.get_secret_value(secret_name="your_secret_name")
+```
+
+This module is also used in other subpackages to simplify the instantiation: the necessary credentials to initialize certain service can be retrieved from AWS Secrets Manager.
+
+#### Amazon Simple Queue Service
+
+Amazon Simple Queue Service (SQS) is a fully managed message queuing service that supports programmatic sending of messages via web service applications as a way to communicate over the Internet.
+
+We provide methods to receive, delete, and send single or a batch of messages.
+
+##### Example of usage
+
+```python
+from aws.sqs_handler import SQSHandler
+
+sqs_handler = SQSHandler()
+entries = [
+    {
+        "Id": "Id1",
+        "MessageBody": "Your message 1",
+    },
+    {
+        "Id": "Id2",
+        "MessageBody": "Your message 1",
+    }
+]
+entries.append(message)
+sqs_handler.send_message_batch(queue_name="your_queue_name", entries=entries)
+```
+
+### Google Cloud Platform
+
+Google Cloud Platform a suite of cloud computing services provided by Google that runs on the same Cloud infrastructure that Google uses internally for its end-user products.
+
+The connection to Google Cloud Platform has been done using the [google-cloud-core](https://googleapis.dev/python/google-api-core/latest/index.html) package.
+
+We provide a handler that allows doing queries on BigQuery services.
+
+The use of this subpackage requires having Google Cloud Platform credentials, these can be passed as arguments in the initializer of the handler class directly, or you can store them in AWS Secrets Manager and retrieve them during the initialization using the `from_aws_secrets_manager` method instead.
+
+##### Examples of initialization

```

Comment:
Since we already gave an example on the secrets manager section, now this can simply be an Example of usage with the init and the query in the same snippet